ci-operator/step-registry/openstack/deprovision/deletefips/openstack-deprovision-deletefips-commands.sh

```diff
@@ -4,8 +4,8 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-echo using ${OS_CLOUD}
-export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
+export OS_CLOUD="$CLUSTER_TYPE"
```

Comment:
We want to standardize on `OS_CLOUD="openstack"`. We can revisit if we decide to have multiple cloud entries in a single `clouds.yaml` file.